packages/config-loader/src/lib/reader.ts

```diff
@@ -68,6 +74,17 @@ export async function readConfigFile(
       }
     }
 
+    const [secretKey] = Object.keys(obj).filter(key => key.startsWith('$'));
```

Comment:
What if there is more than one?

Reply:
It picks the first one. We could throw as well though. Think I'll add a `keys.length === 1` check